pkgs/misc/emulators/tilem/default.nix

```
@@ -0,0 +1,66 @@
+{ stdenv, fetchurl, runCommand, lib, libarchive, autoreconfHook,
+  pkgconfig, glib, libusb1, darwin, acl, lzma, bzip2, gnome2 }:
+
+let
+  tilibs = fetchurl {
```

```suggestion
  tilibs = buildins.fetchTarball {
```
`buildins.fetchTarball` fetches & extracts a tarball, I believe that's what you were looking for? `sha256` will need to be adjusted.